README.md

```diff
@@ -4,76 +4,123 @@
 
-`Red Hat Dependency Analytics (RHDA)` is powered by [Snyk Intel Vulnerability DB](https://snyk.io/product/vulnerability-database/). It is the most advanced and accurate open source vulnerability database in the industry and adds value with the latest, fastest, and numerous vulnerabilities derived from multiple sources.
+With the Red Hat's Dependency Analytics (RHDA) extension you can generate vulnerability reports for your application's dependencies by highlighting security concerns within your software supply chain.
+The Dependency Analytics extension uses the [Snyk Vulnerability Database](https://snyk.io/product/vulnerability-database/) for the most up-to-date vulnerability information available.
+Snyk uses industry-leading security intelligence by pulling from many data sources to give you exact vulnerability information, along with advice on how to remediate those security risks.
+
+**NOTE:**
+<br >The Red Hat Dependency Analytics extension is an online service hosted and maintained by Red Hat.
+Dependency Analytics only accesses your manifest files to analyze your application dependencies before displaying the vulnerability report.
+
+**IMPORTANT:**
+<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), and Node ecosystems (`npm`).
+In future releases, Red Hat plans to support other programming languages.
+
+##### Table of Contents
+- [Quick start](#quick-start)
+- [Configuration](#configuration)
+- [Features](#features)
+- [Using Red Hat Dependency Analytics for CI builds](#using-red-hat-dependency-analytics-for-ci-builds)
+- [Know more about the Red Hat Dependency Analytics platform](#know-more-about-the-red-hat-dependency-analytics-platform)
+- [Data and telemetry](#data-and-telemetry)
+- [Support, feedback \& questions](#support-feedback--questions)
+- [License](#license)
+
+## Quick start
+
+**Prerequisites**
+
+- At least one of the following binaries must exist in your system’s `PATH` environment:
+  - `mvn`
+  - `npm`
+
+  <br >**IMPORTANT:** 
+  <br >Visual Studio Code by default executes binaries directly in a terminal found in your system's `PATH` environment.
+  You can configure Visual Studio Code to look somewhere else to run the necessary binaries.
+  You can configure this by accessing the [extension settings](https://code.visualstudio.com/docs/getstarted/settings).
+  Click the **Workspace** tab, search for the word _executable_, and specify the absolute path to the binary file you want to use for Maven or Node.
+- Optionally, a Snyk token.
+
+**Procedure**
+
+1. Install [Visual Studio Code](https://code.visualstudio.com/) on your workstation.
+2. After the installation finishes, open the Visual Studio Code application.
+3. Click the **Extensions** icon from the **Activity Bar** or click **View** from the file menu, and click **Extensions**.
+4. Search the **Marketplace** for _Dependency Analytics_.
```

Comment:
Good to know the marketplace name will change, but searching for "Dependency Analytics" will return the correct result.

Not sure I understand your second comment, which could be said about anything.  Things will always be changing, but currently those are two ways a user can access the marketplace page to install extension.  We should not  assume the user has knowledge about something, so it's best to be very clear, and concise when writing procedures.

Reply:
Since we have changed the name of the VScode extension from "Dependency Analytics" to "Red Hat Dependency Analytics" we must promote the correct and full name of the extension. If in the future there shall be more extensions including "Dependency Analytics" in the extension name this will bare more results than just one.

Regarding the second comment, in general I recommend to avoid documenting the functionalities of external applications in ones own application, due to the possibility of these functionalities changing in the future. For instance, we cannot go and edit our README and create a new release each time VScode changes a name of a button. It is ok to assume the user has no knowledge about how to access the specific component, therefore it would be best to refer him to the external applications documentation page where possible.  
Or, at least, i do not recommend to list multiple different ways to enter the Extensions marketplace as written, since there are many. It might  be better to only list 1 that is the most obvious and thats least likely to change.

Reply:
Okay, I'll just provide one way into the VS Code marketplace.

Just want to re-emphasize the point, writing good procedures should be prescriptive, and clear, telling the user exactly what to do, and not on assumptions or pointing users to read something else somewhere else.  To be clear, concise, and prescriptive you have to write in context of the application's framework that our extension relies on.  Sure, Microsoft may changes things sometime in the future, but when they do, it's on us to follow through with a timely update to documentation to reflect that change, if it impacts the use of our extension.